### PR TITLE
ROX-17673: Prepare dummy test target for sensor-integration tests

### PR DESCRIPTION
**Pull request by vikin91**
## Description

This PR adds dummy test run (one that always passes) required by https://github.com/openshift/release/pull/40182

The proper moving of the tests will happen in the follow-up 👉 The goal of https://issues.redhat.com/browse/ROX-17674 is to move the sensor integration tests away from the nongroovy step as the integration tests are now larger and keep growing. This would allow us (who work on Sensor) to iterate quicker when adding new features and tests in the upcoming epic.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

## Testing Performed

- Running it locally the way the CI will run this, i.e.:
```
QUAY_RHACS_ENG_RO_USERNAME="xxx" \
QUAY_RHACS_ENG_RO_PASSWORD="yyy" \
ORCHESTRATOR_FLAVOR="openshift" \
MAIN_IMAGE_TAG="4.0.x-619-g2c7a6f32b8" \
KUBECONFIG="~/.cluster/kubeconfig" \
./tests/e2e/sensor.sh
```

### CODE REVIEW

File: tests/e2e/sensor.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Runs all e2e tests. Derived from the workload of CircleCI gke-api-nongroovy-tests.
```
Comment (gavin-stackrox) [inline tests/e2e/sensor.sh:6]:
```suggestion
# Runs sensor integration tests.
```

**Review by gavin-stackrox — APPROVED** [review]
It is a little bit more than a dummy but either way LGTM.

File: tests/e2e/sensor.sh (hunk 1 above)
Comment (vikin91) [reply tests/e2e/sensor.sh:6]: I will merge now to unblock [the other PR](https://github.com/openshift/release/pull/40182), but I will correct this comment in the followup PR.

Comment (vikin91) [reply tests/e2e/sensor.sh:6]: Here it is: https://github.com/stackrox/stackrox/pull/6416

**Pull request merged**